### ISSUE TITLE
processClouds-loop.ipynb: focdistsFromCsv() 

### ISSUE THREAD

**Issue by KrisHeylen** [montesmariana/semcloud#1]
getclouds() for focdists throws error without creating output: 
![image](https://user-images.githubusercontent.com/3693843/133063406-15fc2758-7295-4b63-92e9-c6362ef86143.png)

**Comment by montesmariana** [comment]
ahhhrg that must be the different versions of readr, this is a new argument I think. I haven't gotten around to specifying versions of the packages I'm using. Did you try `update_packages()`?

**Comment by KrisHeylen** [comment]
I `update_packages()` several times already, both from within the jupyter notebook and in a command-line R-session. Of course I cannot update the global ones (no permissions), only the local ones. But I keep bumping into these outdated packages in the kernel, like _rlang_  that I cannot unload to load the newer ones in my local directory that semclouds neesds:

![image](https://user-images.githubusercontent.com/3693843/133087915-29e6ae81-6161-455a-8d4e-59cea370dbc3.png)

Are you sure you using the R-installation in the kernel? Or are you using your own local R version across the board so that can update whatever you want?
Can you check?  The version in the kernel is 3.6.3

![image](https://user-images.githubusercontent.com/3693843/133088192-e31ef90c-c40a-4035-9209-0bb2a72bd8b5.png)

**Comment by montesmariana** [comment]
I have the same output for `version`.
But my libraries are not stored in 'home/mariana/local/R_libs'... what happens if you _don't_ specify where they should be stored?

On the other hand, the times I've had the "is already loaded but X is required", the only solution I have found is deleting the folder with the package and installing it again.

These are the paths and version of some of my packages:

![image](https://user-images.githubusercontent.com/54317904/133095908-c53aecef-9d33-4aaf-b67b-4e5becf66733.png)

**Comment by KrisHeylen** [comment]
Ok. I'll remove all my local R folders and will run `update.packages()` again. fingers crossed that that will work :x

**Comment by montesmariana** [comment]
If you remove the folders you will have to _install_ the packages (there will be no packages to update, as far as I understand)

**Comment by KrisHeylen** [comment]
Update.packages() worked just fine. Thanks!

**Comment by KrisHeylen** [comment]
PS: Not completely sure what I did right this time, but I can guess: Before, I used `install.packages()` to install (newer versions of) specific packages into my local directory and then I loaded the corresponding library in the jupyter R-session specififying my local directory. Although I could then load the library, dependencies couldn't be loaded even though they were also installed locally. I guess  there is an hierarchy of package versions (the _namespace_) that doesn't get updated when installing new packages this way.
The second time, after removing all my local packages, I used `update.packages()` and then got the message I had no permissions to update the global packages and I was offered the option to update then locally, which I did. I guess in this case, the namespace hierarchy is updated so that when loading a library, the newer locally installed dependencies will be loaded as well.
